### ISSUE TITLE
pi_logit must indicate logits / pi must indicate probability distributions

### ISSUE THREAD

**Issue by rhaps0dy** [AlignmentResearch/polygames#17]
title

**Comment by rhaps0dy** [comment]
Fixed in #21
